fedimint-server/src/net/connect.rs

```diff
@@ -232,12 +237,13 @@ pub fn dns_sanitize(name: &str) -> String {
 }
 
 /// Parses the host and port from a url
-pub fn parse_host_port(url: &SafeUrl) -> anyhow::Result<String> {
+pub fn parse_host_port(url: &SafeUrl, fallback_port: Option<u16>) -> anyhow::Result<String> {
     let host = url
         .host_str()
         .ok_or_else(|| format_err!("Missing host in {url}"))?;
     let port = url
         .port()
```

Comment:
There's already `SafeUrl::port_or_known_default`

Reply:
Totally forgot about that, thx!